monkey/monkey_island/cc/services/bootloader.py

```diff
@@ -0,0 +1,69 @@
+from typing import Dict, List
+
+from bson import ObjectId
+
+from monkey_island.cc.database import mongo
+from monkey_island.cc.services.node import NodeService, NodeNotFoundException
+from monkey_island.cc.services.utils.node_groups import NodeGroups
+from monkey_island.cc.services.utils.bootloader_config import SUPPORTED_WINDOWS_VERSIONS, MIN_GLIBC_VERSION
+
+
+class BootloaderService:
+
+    @staticmethod
+    def parse_bootloader_telem(telem: Dict) -> bool:
```

Comment:
rename the function. The parsing is done in the `parse_bootloader_request_windows` or `parse_bootloader_request_linux` functions. A good direction for a name to this function is `determine if monkey can run` or `is operating system supported`

Reply:
The problem is that `parse_bootloader_telem` is also responsible for creating nodes, adding bootloader telem to the database and so on. It does parse the data of telemetry, whilst `parse_bootloader_request_windows` only extract the data from request into json. I renamed the latter, maybe it will make more sense.